packages/paste-icons/__IconList.tsx

```diff
@@ -3,19 +3,21 @@ import * as React from 'react';
 import {Text} from '@twilio-paste/text';
 import {Grid, StoryIcon} from './__StoryStyles';
 
+import {CopyIcon} from './src/CopyIcon';
 import {LoadingIcon} from './src/LoadingIcon';
 import {PlusIcon} from './src/PlusIcon';
 
 interface IconProps {
   title?: string;
-  decorative?: boolean;
```

Comment:
I followed the instructions outlined here: https://github.com/twilio-labs/paste/blob/master/packages/paste-icons/README.md#how-can-i-contribute-icons

Interesting the tool removed the `?`

Reply:
Yup this change is good.  The `?` there was an oversight.